api/app/Models/TestEntity.php

```diff
@@ -38,13 +43,38 @@ class TestEntity extends BaseModel
         'updated_at' => 'dateTime',
     ];
 
+    public function addSecondTest(SecondTestEntity $entity)
+    {
+        $this->testMany->add($entity);
+        return $this;
+    }
+
     /**
-     * Get the access route for the entity.
-     *
-     * @return string
+     * @return array
      */
-    public function entityRoute()
+    public function getValidationRules()
+    {
+        return [
```

Comment:
As the validation rules are almost always a static array, they should be defined in the model rather than a method. The `getValidationRules()` method should just return that model property unless the child class overrides it.